lib/preload_counts/ar.rb

```diff
@@ -63,7 +63,15 @@ def scope_to_select(association, scope)
 
       r_scope = self.reflections.with_indifferent_access[association].scope
       if r_scope
-        conditions += self.instance_eval(&r_scope).where_values
+        where_values = self.instance_eval(&r_scope).where_values
+        where_values.each do |where_value|
+          if where_value.respond_to? :to_sql
+            sql_value = where_value.to_sql
+            conditions << sql_value unless sql_value.include?('$1') # ignoring parameterized clauses
```

Comment:
`to_sql` can return parameterized where clauses like "table.id = $1" which will be ignored for now.